src/libs/ReportUtils.js

```diff
@@ -1304,6 +1305,82 @@ function getReportPreviewMessage(report, reportAction = {}) {
     return Localize.translateLocal('iou.payerOwesAmount', {payer: payerName, amount: formattedAmount});
 }
 
+/**
+ * Get the report action message when expense has been modified.
+ *
+ * @param {Object} reportAction
+ * @returns {String}
+ */
+function getModifiedExpenseMessage(reportAction) {
+    const reportActionOriginalMessage = lodashGet(reportAction, 'originalMessage', {});
+    if (!_.isEmpty(reportActionOriginalMessage)) {
+        if (reportActionOriginalMessage.oldAmount &&
+            reportActionOriginalMessage.oldCurrency &&
+            reportActionOriginalMessage.amount &&
+            reportActionOriginalMessage.currency
```

Comment:
NAB: I think we should extract this to a variable for readability. Similar stuff for below

```js
const hasModifiedAmountAndCurrency = reportActionOriginalMessage.oldAmount &&
            reportActionOriginalMessage.oldCurrency &&
            reportActionOriginalMessage.amount &&
            reportActionOriginalMessage.currency
```